readme.md

```diff
@@ -78,6 +78,17 @@ Overrides `projectName`.
 
 The only use-case I can think of is having the config located in the app directory or on some external storage.
 
+#### encryptionKey
+
+Type: `string | Buffer | TypedArray | DataView`<br>
+Default: `undefined`
+
+Note that this **is not intended for security purposes**, since the encryption key would be easily found inside a plain-text NodeJS application.
+
+Its main use is for obscurity: if a user was looking through the user config directory and found the config file, since it's a simple json file they may be tempted to modify/add/remove a key/value pair. By providing an encryption key `Conf` will obfuscate the file which should hopefully deter any users from doing so.
+
+It also has the added bonus that it will maintain a config file's integrity: if the file is changed in any way then decryption will not work, in which case the store will just reset back to its default state.
+
```

Comment:
Can you include that it's encrypted with `aes-256-cbc` and linkfy the name with a resource about this encryption.

Reply:
I was thinking about this, and I think we should change the options from:

    new Conf({ encryptionKey: 'my_key' });

To something like:

    new Conf({ encryption: { key: 'my_key', algorithm: 'aes-256-cdc' });

That way the user can choose the algorithm. Since [the algorithm is dependent on OpenSSL](https://nodejs.org/api/crypto.html#crypto_crypto_createcipher_algorithm_password) it may be useful for some systems whose OpenSSL doesn't have `aes-256-cbc`. (I'm not familiar with different installations of OpenSSL, so I don't know if they do or don't have this style of encryption).

---

And if `encryption` is just a string, then fallback to just using that as the encryption key with a sensible default.

Reply:
Nah, I prefer to keep it simple.

I always keep this in the back of my head when designing APIs: 😛

![horrible-ui](https://user-images.githubusercontent.com/170270/30054843-a3bd5f6c-9257-11e7-8a6a-5ed546d1be92.png)